samples/storage_csm_powerscale.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: test-isilon
   namespace: test-isilon
 spec:
+  remoteRepo:
+    repository: "https://amaas-eos-mw1.cec.lab.emc.com:5036/artifactory/csi-driver-helm-virtual"
```

Comment:
@yifeijin, we should not be having references to any of the internal repo
FYI @bharathsreekanth